tensorflow/python/layers/core.py

```diff
@@ -57,6 +57,9 @@ class Dense(base.Layer):
       linear activation.
     use_bias: Boolean, whether the layer uses a bias.
     kernel_initializer: Initializer function for the weight matrix.
+      If `None` (default), weights are initialized using the global default
+      initializer which uses the `glorot_uniform_initializer` defined in
+      `tensorflow.python.ops.init_ops`.
```

Comment:
Thanks for the comment, and I completely agree. I was looking for a non-internal path to the `glorot_uniform_initializer` when I read this a while ago but couldn't find any except `tf.contrib.keras.initializers.glorot_uniform`. So, would this the one that you recommend here, or is there an equivalent similar to `tf.truncated_normal_initializer` (i.e., is the glorot/xavier initializer that is in the `layers` submodule, or is this going to be moved from `tf.contrib.keras.initializers.glorot_uniform` to `tf.layers.initializers.glorot_uniform` later?). What would be your recommendation on this?